src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,136 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private StorageDao storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUP() {
+        storageDao = new StorageDaoImpl();
+    }
+
+    @Test
+    void register_loginLessThanMinLength_notOk() {
+        User user = new User();
+        user.setLogin("Bob");
+        user.setPassword("ghjkl543");
+        user.setAge(23);
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(user);
+        });
```

Comment:
The test method `register_loginLessThanMinLength_notOk` does not follow the checklist's requirement to check all edge cases. It should include tests for the exact minimum length and one less than the minimum length.